### PR TITLE
add fulltext-index after column creation

### PR DESCRIPTION
**Pull request by HenryLamorski**
fulltext index after column-creation.

### CODE REVIEW

File: src/MetaModels/Attribute/Longtext/Longtext.php [hunk 1]
```diff
@@ -38,6 +41,22 @@ public function getSQLDataType()
...
     /**
+     * @{inheritDoc}
+     *
+     * add a fulltext-index to column after creation
+     */
+    public function createColumn()
+    {
+        parent::createColumn();
+
+        TableManipulation::addIndex(
```
Comment (discordier) [inline src/MetaModels/Attribute/Longtext/Longtext.php]: Do we have to drop this index prior deleting the column?
If so, we will need to override the deleteColumn() as well.

**Comment by discordier** [comment]
@xantippe have you checked this out? Can you create and delete a longtext attribute after this change?

File: src/MetaModels/Attribute/Longtext/Longtext.php (hunk 1 above)
Comment (HenryLamorski) [inline src/MetaModels/Attribute/Longtext/Longtext.php]: no, the Database update the index on drop/alter querys automatically.

On 03.03.2016 09:01, Christian Schiffler wrote:
> > In src/MetaModels/Attribute/Longtext/Longtext.php
<https://github.com/MetaModels/attribute_longtext/pull/12#discussion_r54846508>:
> > > @@ -38,6 +41,22 @@ public function getSQLDataType() > >      } >
>  > >      /** > > +     * @{inheritDoc} > > +     * > > +     * add a
fulltext-index to column after creation > > +     */ > > +    public
function createColumn() > > +    { > > +        parent::createColumn();
> > + > > +        TableManipulation::addIndex( > > Do we have to drop
this index prior deleting the column? > If so, we will need to override
the deleteColumn() as well. > > — > Reply to this email directly or view
it on GitHub
<https://github.com/MetaModels/attribute_longtext/pull/12/files#r54846508>.
>

-- 
--
Henry

**Comment by zonky2** [comment]
pls look at https://github.com/MetaModels/filter_text/pull/22#issuecomment-242542244

**Pull request closed without merging**